app/helpers/goals_helper.rb

```diff
@@ -0,0 +1,11 @@
+module GoalsHelper
+  def time_left_until_expires(goal)
```

Comment:
Consider testing `time_left_until_expires(goal)` in a helper spec. Here's an example of a helper spec: https://github.com/eliotsykes/rspec-rails-examples/blob/master/spec/helpers/application_helper_spec.rb